Samples/SwiftKitSampleApp/src/main/java/com/example/swift/HelloJava2Swift.java

```diff
@@ -15,58 +15,55 @@
 package com.example.swift;
 
 // Import swift-extract generated sources
-import com.example.swift.generated.ExampleSwiftLibrary;
 import com.example.swift.generated.MySwiftClass;
 
 // Import javakit/swiftkit support libraries
+import org.swift.swiftkit.SwiftArena;
 import org.swift.swiftkit.SwiftKit;
-
+import org.swift.swiftkit.SwiftValueWitnessTable;
```

Comment:
Moved some functionality out of SwiftKit, keeping it for things end users might wanna use.